am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -57,8 +57,11 @@ void filterResource_whenRowNotExistWithAccessorIdAndResourceId_ReturnNull() {
     }
 
     @Test
-    void filterResource_whenRowExistsAndDoesntHaveReadPermissions_ReturnNull() {
-        ams.createResourceAccess(createRecord(resourceId, ACCESSOR_ID, Stream.of(CREATE, UPDATE).collect(toSet())));
+    void whenRowExistsAndDoesntHaveReadPermissionsReturnNull() throws TransactionRolledbackException {
+        Map<JsonPointer, Set<Permission>> rootLevelCreatePermission = new ConcurrentHashMap<>();
+        rootLevelCreatePermission.put(JsonPointer.valueOf("/"), EXPLICIT_CREATE_PERMISSION);
```

Comment:
Test above uses same path but variable name is different. What is more `/` does not mean root level. Let's correct name. For me `attributePermission` would be good enough.